staging/dex-k8s-authenticator/Chart.yaml

```diff
@@ -2,14 +2,10 @@ apiVersion: v1
 appVersion: "v1.1.0"
 description: "Authenticator for using Dex with Kubernetes"
 name: dex-k8s-authenticator
-version: 1.1.3
+version: 1.1.4
+home: https://github.com/mesosphere/charts
 sources:
 - https://github.com/mintel/dex-k8s-authenticator
 maintainers:
-- name: Martin Hrabovcin
+- name: mhrabovcin
   email: mhrabovcin.c@mesosphere.com
-# Original maintainers from `mintel/dex-k8s-authenticator` repository
```

Comment:
I had to remove these maintainers bc the lint check was complaining that the names weren't associated to usernames in gh.